breacharbiter.go

```diff
@@ -8,6 +8,8 @@ import (
 	"io"
 	"sync"
 
+	"github.com/lightningnetwork/lnd/labels"
+
```

Comment:
Nit: remove newline to have the imports grouped. There are a few other cases throughout the diff.